Automattic/jetpack
# Carousel: Try replacing light theme styles with CSS variables

Fixes #

This PR explores the idea of replacing the specific `.jp-carousel-light` styles for the Carousel with CSS variables to simplify the CSS we're delivering to the browser.

~It's currently a work in progress — the main area that could use some tweaking is the commenting area, at the moment we still have a number of separate colors being used, which makes it a little harder to come up with a clean set of just a few variables. If we proceed with this PR, we might want to tighten up the styles there a bit~ — Update: we've tweaked the commenting area and simplified some of the colors a bit to reduce variation and consolidate where possible. Happy to continue tweaking if it looks off to anyone!

#### Screenshot

Updated light mode view of the comment form:

![image](https://user-images.githubusercontent.com/14988353/127793101-775b50ce-e62c-4af7-8f45-0ae8ba26ea48.png)

#### Changes proposed in this Pull Request:

* Replace Carousel light theme styles with CSS variables
* Consolidate some of the CSS colors to better align with a small list of CSS variables
* Roll in comments indicator color inversion change from https://github.com/Automattic/jetpack/pull/20514 and switch to use CSS variables

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Enable the Carousel module and insert a gallery block into a post
* Switch between light and dark mode and test the carousel's styles, ensure that things are roughly the same as, or improved on the styling that currently exists
* Test across browsers to ensure that nothing breaks
